planning/behavior_path_planner/include/behavior_path_planner/scene_module/scene_module_interface.hpp

```diff
@@ -322,6 +329,8 @@ class SceneModuleInterface
 
   PlanResult getPathReference() const { return path_reference_; }
 
+  MarkerArray getInfoMarkers() { return info_marker_; }
```

Comment:
Should add const for getter functions
No need to fix these in this PR since there are some lines to fix.

Reply:
You're right. I'll fix.

Reply:
fixed